go/staking/api/api.go

```diff
@@ -88,6 +94,13 @@ var (
 		MethodAmendCommissionSchedule,
 	}
 
+	// ContextKeyTokenSymbol is the key to retrieve the token's ticker symbol
+	// value from a context.
+	ContextKeyTokenSymbol = contextKey("staking/token-symbol")
```

Comment:
The `Context` prefix sounds too general as we have other kinds of contexts. Maybe explicitly name this `PrettyPrinterContext...`.

Also consider moving all this pretty printing stuff into its own `prettyprint.go` file.

Reply:
> The `Context` prefix sounds too general as we have other kinds of contexts. Maybe explicitly name this `PrettyPrinterContext...`.

Yes, good point. I've renamed it.

> Also consider moving all this pretty printing stuff into its own `prettyprint.go` file.

Did you mean just the new `ConvertToTokenAmount()` and `PrettyPrintAmount()` functions?

For `PrettyPrint()` and `PrettyType()` methods, I would rather have them defined in the same place as the corresponding types.

If we would like to make the `go/staking/api/api.go` file smaller, I would rather break out logical pieces (e.g. events-related stuff, transaction-related stuff, threshold-related stuff, or similar).

Reply:
I mean just the new functions.

Reply:
> I mean just the new functions (and context keys).

Aha, ok. I can move those to `prettyprint.go`.

Reply:
> > I mean just the new functions (and context keys).
> Aha, ok. I can move those to `prettyprint.go`.

Done.